### PR TITLE
Update contour, which no longer depends on geojson crate

### PR DESCRIPTION
**Pull request by michaelkirk**
No change in behavior here for abstreet, I just go tired of having to update contour every time there was a geojson release.

### CODE REVIEW

File: apps/ltn/src/draw_cells.rs [hunk 1]
```diff
@@ -220,23 +220,19 @@ impl RenderCellsBuilder {
...
-                    geojson::Value::MultiPolygon(polygons) => {
-                        for p in polygons {
-                            if let Ok(poly) = Polygon::from_geojson(&p) {
-                                cell_polygons.push(
-                                    poly.must_scale(RESOLUTION_M)
-                                        .translate(self.bounds.min_x, self.bounds.min_y),
-                                );
-                            }
-                        }
+            for contour in contour_builder.contours(&grid.data, &thresholds).unwrap() {
```
Comment (michaelkirk) [inline apps/ltn/src/draw_cells.rs:227]: I was able to test and run all the other changes, but I wasn't sure how to get to this for the ltn app. So the changes in this file are untested.

File: map_gui/src/tools/heatmap.rs [hunk 2]
```diff
@@ -196,27 +196,18 @@ pub fn make_heatmap(
...
-                    // Don't block the map underneath
-                    let color = Color::rgb(c.r as usize, c.g as usize, c.b as usize).alpha(0.6);
-                    for p in polygons {
-                        if let Ok(poly) = Polygon::from_geojson(&p) {
-                            batch.push(color, poly.must_scale(opts.resolution));
-                        }
-                    }
+        let contour_builder = contour::ContourBuilder::new(grid.width as u32, grid.height as u32, false);
+        for contour in contour_builder.contours(&grid.data, &thresholds).unwrap() {
+            let (geometry, threshold) = contour.into_inner();
```
Comment (michaelkirk) [inline map_gui/src/tools/heatmap.rs:201]: Each contour records its threshold, so there's no need for the previous `zip`.

File: apps/game/src/layer/elevation.rs [hunk 3]
```diff
@@ -294,27 +294,23 @@ impl ElevationContours {
...
+            let contours = contour_builder.contours(&grid.data, &thresholds).unwrap();
             timer.stop("calculate contours");
 
-            timer.start_iter("draw", features.len());
-            for (feature, color) in features.into_iter().zip(colors) {
+            timer.start_iter("draw", contours.len());
+            for (contour, color) in contours.into_iter().zip(colors) {
                 timer.next();
-                match feature.geometry.unwrap().value {
-                    geojson::Value::MultiPolygon(polygons) => {
```
Comment (dabreegster) [inline apps/game/src/layer/elevation.rs:305]: Much nicer API

File: apps/ltn/src/draw_cells.rs (hunk 1 above)
Comment (dabreegster) [reply apps/ltn/src/draw_cells.rs:227]: I tried them out, everything looks the same! For the record, you could run the LTN app, click any neighbourhood, and just see it split into colored "cell" areas

**Review by dabreegster — APPROVED** [review]
Thanks for the PR and the underlying work improving `contour`! Much nicer API and less maintenance

**Pull request merged**